mcstatus/protocol/connection.py

```diff
@@ -10,19 +10,19 @@
 from ctypes import c_uint32 as unsigned_int32
 from ctypes import c_uint64 as unsigned_int64
 from ipaddress import ip_address
-from typing import Optional, TYPE_CHECKING, Union
+from typing import TYPE_CHECKING
 
 import asyncio_dgram
 
 from mcstatus.address import Address
 
 if TYPE_CHECKING:
-    from typing_extensions import SupportsIndex  # Python 3.7 doesn't support this yet.
+    from typing_extensions import SupportsIndex, TypeAlias  # Python 3.7 doesn't support `SupportsIndex` yet.
```

Comment:
This comment doesn't seem too necessary, we use `if TYPE_CHECKING` a lot in the codebase and people will most likely be familiar with what it's doing. I like to keep the codebase consistent and I don't believe we use comments like these in many other places (if any), so let's remove it from here too.

Reply:
Fixed in 190aa9d5682c8ee93bb2d0b0d61ae4990e4c4306.